dev/tests/integration/testsuite/Magento/GraphQl/_files/product_downloadable_with_purchased_separately_links_rollback.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+use Magento\TestFramework\Helper\Bootstrap;
+use Magento\Framework\Exception\NoSuchEntityException;
+
+/** @var \Magento\Framework\Registry $registry */
+$registry = Bootstrap::getObjectManager()->get(\Magento\Framework\Registry::class);
+
+$registry->unregister('isSecureArea');
+$registry->register('isSecureArea', true);
+
+/** @var \Magento\Catalog\Api\ProductRepositoryInterface $productRepository */
+$productRepository = Bootstrap::getObjectManager()
+    ->get(\Magento\Catalog\Api\ProductRepositoryInterface::class);
+
+try {
+    $product = $productRepository->get(
+        'graphql-downloadable-product-with-purchased-separately-links',
+        false,
+        null,
+        true
+    );
+    $productRepository->delete($product);
+} catch (NoSuchEntityException $e) {
```

Comment:
Need to provide comment in empty catch like:
```
/**
  * Tests which are wrapped with MySQL transaction clear all data by transaction rollback
  */
```

Reply:
**added**